src/components/badge/examples/badge.scss

```diff
@@ -0,0 +1,3 @@
+:host(limel-example-badge) {
```

Comment:
woops
<img width="756" alt="Screenshot 2023-02-21 at 18 35 51" src="https://user-images.githubusercontent.com/35954987/220418934-c0528d66-059c-4446-b024-ceef95481dd6.png">

I think this is a buggy behaviour which we have never noticed before. Because all previous examples had `display: flex` on the host.

The fix for it is to go to `badge.scss` and change `display: flex` to `display: inline-flex`. This is more correct and prevents such eff-ups.

Can you add please a new commit to this PR Victor?

Reply:
`fix(badge): prevent stretching through the width of block containers`

Reply:
<img width="1369" alt="Screenshot 2023-02-22 at 09 19 55" src="https://user-images.githubusercontent.com/35954987/220562739-ea5b9326-1cc5-4520-a1d2-daade027659c.png">

Reply:
Good catch! I will do that

Reply:
64fa53dc068afbb1d96d320f0b2fe8a650f19320